cms_coding_rules.md

```diff
@@ -88,10 +88,15 @@ If necessary to create a unique name, one can add the directory name:
 27. Provide meaningful argument names in method declarations in the header ﬁle to indicate usage, unless the type fully describes the usage.
 28. Try to avoid excessively long lines of code that impair readability.
 29. Data members of a class must not be redeﬁned in derived classes since doing so hides the original data member and could create confusion between the original and redefined data members.
+30. Use `std::abs()` instead of `fabs()`.
+31. Use C++ headers, e.g. `#include <cmath>` instead of `#include <math.h>`.
+32. Use MessageLogger instead of `std::cout` and `std::cerr` (\*). Allowed exception: standalone programs should use `std::cout` and `std::cerr`.
```

Comment:
Then please *fix* the message logger poor usability.
Enabling debugging messages for individual modules is at best a pain.